cfn/paws-collector.template

```diff
@@ -606,6 +616,12 @@
                   "paws_max_pages_per_invocation": {
                     "Ref":"PawsMaxPagesPerInvocation"
                   },
+                  "DD_API_KEY": {
```

Comment:
could you please move it on top, right after `aws_lambda_update_config_name`. I'd like to keep paws parameters close to each other.